### ISSUE TITLE
Resolving subtypes does not work

### ISSUE THREAD

**Issue by strahlex** [jstasiak/python-zeroconf#37]
Resolving mdns subtypes does not work because of this check: https://github.com/jstasiak/python-zeroconf/blob/master/zeroconf.py#L1085

E.g. `browser = ServiceBrowser(zeroconf, "_config._sub._machinekit._tcp.local.", listener)` returns following match: 
```
$ avahi-browse _launcher._sub._machinekit._tcp
+   eth0 IPv4 Machinekit Launcher on jessie-aroessler.local _machinekit._tcp     local
```

Meaning that if a subtype is used the type does not show up in the resulting response.

**Comment by strahlex** [comment]
@jstasiak Can you please take a look at this. DNS subtypes are part of the [DNS-SD](https://tools.ietf.org/html/rfc6763) specification.

**Comment by stephenrauch** [comment]
Closed via #53
